src/workerd/io/worker.c++

```diff
@@ -2196,7 +2192,19 @@ void Worker::Lock::validateHandlers(ValidationErrorReporter& errorReporter) {
         report(getEntrypointName(entry.key), entry.value);
       }
       for (auto& entry: worker.impl->actorClasses) {
-        errorReporter.addHandler(getEntrypointName(entry.key), "class");
+        KJ_IF_SOME(entrypointName, getEntrypointName(entry.key)) {
+          errorReporter.addActorClass(entrypointName);
+        } else {
+          // Hmm, it appears someone tried to export a Durable Object class as a default
+          // entrypoint. This doesn't actually work: the runtime will not allow this DO class
+          // to be used, either for actors or as an entrypoint.
+          //
+          // TODO(someday): Make this a hard error. I'm hesitant to do it in my current change
+          //   for fear that it'll break someone somewhere forcing a rollback. For now we log.
+          LOG_PERIODICALLY(ERROR,
```

Comment:
Nit: log periodically is nice but still might be better to make this skip sentry. It is easy enough to query kibana directly for it.

Reply:
I think it'll be fine as I don't think it's even possible to deploy a worker with this problem, but if it's possible, it doesn't accomplish anything.

Don't like using NOSENTRY as there's a big risk of just not ever noticing it at all.

Reply:
heh, I think I'm probably the only one who regularly searches kibana for the NOSENTRY logs we do happen to add ;-)